dsaps/models.py

```diff
@@ -20,13 +21,14 @@ def __init__(self, url):
         logger.info('Initializing client')
 
     def authenticate(self, email, password):
+        """Authenticate user to API."""
         data = {'email': email, 'password': password}
         header = {'content-type': 'application/json', 'accept':
                   'application/json'}
-        session = requests.post(self.url + '/rest/login', headers=header,
+        session = requests.post(f'{self.url}login', headers=header,
```

Comment:
I think what I'd do here is, back up in line 20:
```python
self.url = url.rstrip('/')
```
Then anytime you use `self.url` in this class you would use, for example:

```python
f'{self.url}/login'
```

The reason I suggest removing the trailing slash is because it would be pretty common for the url to be passed both with a trailing slash and without, and this class should be able to handle both. I suggest removing the slash, rather than adding it because I think `f'{self.url}/login` is easier to see as a URL than `f'{self.url}login`. That's mostly preference, but if you decide to stick with the latter, you should still normalize the url:
```python
self.url = url.rstrip('/') + '/'
```

Reply:
That totally makes sense, thanks!